packages/mongodb-rag-core/src/contentStore/MongoDbEmbeddedContentStore.ts

```diff
@@ -234,3 +248,40 @@ export function makeMongoDbEmbeddedContentStore({
     },
   };
 }
+
+type VectorSearchFilter = {
```

Comment:
```suggestion
type MongoDbAtlasVectorSearchFilter = {
```

nit: to make clear that this is for mongodb specifically